testing/tests/DevExpress.ui.widgets.dataGrid/rowsView.tests.js

```diff
@@ -143,30 +143,6 @@ QUnit.test('Create col elements by columns collection', function(assert) {
     assert.equal(cols[4].style.width, '91px', '5 column width');
 });
 
-QUnit.test('Check WAI-ARIA cell labels', function(assert) {
```

Comment:
It is better not to remove the test, but to change it to the area-colindex and selected attributes, There are no other tests for these.

Reply:
Such kind of tests already exists: rowsView.tests.js (Check WAI-ARIA attributes for freeSpace rows/cells after render rows)